src/markdown-pages/nr1-cli/nr1-cli-revised.mdx

```diff
@@ -0,0 +1,92 @@
+---
+path: '/explore-docs/nr1-cli'
+title: 'New Relic One CLI reference'
+template: 'GuideTemplate'
+description: 'An overview of the CLI to help you build, deploy, and manage New Relic apps.'
+tileShorthand:
+  title: 'New Relic One CLI reference'
+  description: 'Learn commands to build and enhance apps'
+redirects:
+  - /build-tools/new-relic-one-applications/cli
+---
+
+<Intro>
+
+To build a [New Relic One app](/build-apps/build-hello-world-app), you must install the New Relic One CLI. The CLI helps you build, publish, and manage your New Relic app.
+
+</Intro>
+
+We provide a variety of tools for building New Relic One apps, including the New Relic One CLI (command line interface). This page will explain how to use [CLI commands](#new-relic-one-cli-commands) to:
+
+- Generate Nerdpack/Nerdlet templates
+
+- Locally serve Nerdpacks (when developing)
+
+- Publish and deploy
+
+- Subscribe to Nerdpacks
+
+- Add screenshots and metadata to the New Relic One Catalog
+
+## Installing the New Relic One CLI
+
+From _within_ New Relic, go to the [**Build your own application** launcher](https://one.newrelic.com/launcher/developer-center.launcher) and follow the instructions. This launcher will automatically generate an API key for the account you select, and give you the pre-populated commands to create a profile, generate your first "Hello World" app, and serve it locally.
+
+![Build New Relic One application](../../images/developercenter.png)
+
+## New Relic One CLI Commands
+
+This table gives descriptions for some of the most important CLI commands. For more context, including usage and option details, click any individual command or the command category. 
```

Comment:
This makes it sound like there are other commands, that aren't as important, but aren't these all the commands?

Reply:
There are aliases we didn't mention, but I agree that's a bit misleading. I'll change to "This table gives descriptions for all of our important CLI commands."